.eslintrc.js

```diff
@@ -12,6 +12,12 @@ module.exports = {
     "prettier/standard",
   ],
 
+  parser: "babel-eslint",
```

Comment:
Not a dependency.

Reply:
I want to use the legacy decorators syntax.